### ISSUE TITLE
Add support for negative margin/padding values

### ISSUE THREAD

**Issue by daohoangson** [daohoangson/flutter_widget_from_html#509]

**Comment by hpoul** [comment]
That would be great. e.g. currently `p` has 1em on top/bottom and `h3` as well. I would like to have `h3` a `margin-bottom` of `-0.5em` to visually bring the following paragraph closer to the headline. I guess this is not possible right now?
(My current solution is to make `p` only `0.5em` vertically and have `h3` with `margin-top: 1em, margin-bottom: 0.5em`, but would be nicer with negative values i think)

**Comment by daohoangson** [comment]
Yeah, negative value would work for your use case. Thanks for the vote, it'll help to prioritize future work.
